tests/executor/test_payment_platforms.py

```diff
@@ -0,0 +1,174 @@
+"""Unit tests for code that selects payment platforms based on driver/network specification."""
+import os
+from unittest import mock
+
+import pytest
+import ya_payment
+
+import yapapi
+from yapapi.executor import Executor, NoPaymentAccountError, DEFAULT_NETWORK, DEFAULT_DRIVER
+from yapapi.rest.payment import Account
+
+
+@pytest.fixture(autouse=True)
+def _set_app_key():
+    os.environ["YAGNA_APPKEY"] = "mock-appkey"
+
+
+def _mock_account(driver: str, network: str, **kwargs) -> Account:
```

Comment:
this would be exactly the place where we could use the factory provided by factory_boy ;)

Reply:
I've simplified test setup a bit in https://github.com/golemfactory/yapapi/pull/215/commits/652312585158193b0c0a7f2e18556a8af7ab2040